projects/index.html

```diff
@@ -1,11 +1,11 @@
 ---
-title: Projects
-excerpt: "This is where we keep a collection of some of the projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
+title: Project Showcase
+excerpt: "This is where we keep a collection of some of the most interesting projects that people are working on using ev3dev. We invite you to click through the links below to see what cool stuff ev3dev can do!"
```

Comment:
> Maybe add a project or two that could be reproduced by a beginner (like demos from one of the libraries), and also introduce classification of the projects by level (beginner, intermediate, advanced)?

I've been debating with myself over how this could be done best. On one hand, the existing projects page could make sense to be used for this; we just mark all (or most of) the existing projects as "Advanced" and our problem is solved. On the other hand, it could make sense to have a separate"examples" page and specify that the projects are a showcase while the examples are things you can try at home. I think I'm leaning in the direction of the former.

>  I like how on lego mindstorms page they have several 'official' robots that may be easily reproduced by a newcomer, and then they have a gallery of community projects. We could even provide programs for those stock robots (and build instructions are already available on lego.com).

Assuming that they license those models in a way that we could use them (with attribution of course), I think that would be great. We could also include the simplest ones like the demo bot that Laurens created. That would, of course, mean that one of us has to build and test the elephant :laughing:! But yes, I think that it would be great to provide source code for these things. And doing it in multiple languages would be very cool! We currently lack good end-to-end examples like that.